modules/programs/vscode.nix

```diff
@@ -78,11 +78,19 @@ in {
             };
 
             when = mkOption {
-              type = types.str;
-              default = "";
+              type = types.nullOr (types.str);
```

Comment:
This may be controversial, but I'd avoid putting in keys with "default" values if that's something users wouldn't do in their own, native keybinding files.

Reply:
Agreed, don't know why I didn't do it this way back then.

Reply:
:)